examples/nextjs-blog/components/header.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * This is a placeholder component - everything can be changed
+ */
+import React, { useState, useEffect } from 'react'
+import { flamelinkApp } from '../utils/flamelink'
+
+const Header = () => {
+  const [mainNav, setMainNav] = useState(null)
+  const [navError, setNavError] = useState(null)
+
+  useEffect(() => {
+    ;(async () => {
```

Comment:
is the `;(async` correct here

Reply:
yup, auto-added by linting tools. It is because we're not using semi-colons anywhere else.

I would've wanted to make the callback function for `useEffect` async but it is not allowed to return a Promise.

Reply:
cools